Server/Quality_of_Life_changer.Contracts/Exceptions/InvalidInputException.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Quality_of_Life_changer.Contracts.Exceptions;
+
+public class InvalidInputException : Exception
+{
+    public InvalidInputException() { }
```

Comment:
rename to `ValidationException` or something similar.
`Input` is confusing here